earth_enterprise/src/fusion/autoingest/sysman/khAssetManager.cpp

```diff
@@ -574,8 +574,12 @@ khAssetManager::NotifyVersionStateChange(const std::string &ref,
 {
   // assert that we're already locked
   assert(!mutex.TryLock());
+  notify(NFY_VERBOSE, "Asserted that mutex is already locked");
```

Comment:
Assert is noisy enough that you probably don't need to log that the assert succeeded